app/pods/settings/overview/template.hbs

```diff
@@ -1,33 +1,39 @@
 <div class="vl-u-spacer-extended-l vl-u-display-flex vlc-u-flex-wrap">
   <div class="vl-u-spacer-extended-s">
-    <button data-test-manage-goverment-domains
-            type="button"
-            class="vl-button vl-button--tertiary vl-button--icon-before"
-      {{action "toggleProperty" "isAddingMandate"}}
+    <LinkTo
+      data-test-manage-government-domains
```

Comment:
ge hebt deze testselector een andere naam gegeven, aangepast in de selectors file ?
testen falen erop